src/main/java/org/apache/flink/olap/benchmark/HighAvailabilityServiceBenchmark.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.olap.benchmark;
+
+import org.apache.curator.test.TestingServer;
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.benchmark.BenchmarkBase;
+import org.apache.flink.benchmark.FlinkEnvironmentContext;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.configuration.HighAvailabilityOptions;
+import org.apache.flink.runtime.jobgraph.JobGraph;
+import org.apache.flink.runtime.jobgraph.JobVertex;
+import org.apache.flink.runtime.jobmanager.HighAvailabilityMode;
+import org.apache.flink.runtime.testtasks.NoOpInvokable;
+import org.apache.flink.util.FileUtils;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.VerboseMode;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.UUID;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.openjdk.jmh.annotations.Scope.Thread;
+
+/** The benchmark for submitting short-lived jobs with and without high availability service. */
+@OutputTimeUnit(SECONDS)
+public class HighAvailabilityServiceBenchmark extends BenchmarkBase {
+	public static void main(String[] args) throws RunnerException {
+		Options options =
+				new OptionsBuilder()
+						.verbosity(VerboseMode.NORMAL)
+						.include(".*" + HighAvailabilityServiceBenchmark.class.getCanonicalName() + ".*")
+						.build();
+
+		new Runner(options).run();
+	}
+
+	@Benchmark
+	public void submitJobThroughput(HighAvailabilityContext context) throws Exception {
```

Comment:
This method will submit a job in each invocation and the default value of `OperationsPerInvocation` is `1`. This is what we need, so there is no need to specify the `@OperationsPerInvocation` value

Reply:
Don't we need a few runs to work around the volatility of run times? ...or am I missing something here? :thinking:

Reply:
@XComp From my understanding of [OperationsPerInvocation](https://javadoc.io/static/org.openjdk.jmh/jmh-core/1.21/org/openjdk/jmh/annotations/OperationsPerInvocation.html), it is used to claim the how many operation the invocation exactly contains. In this case, we measure the job submitting operation and there is only 1 operation per invocation.

Reply:
Regarding the volatility, the BenchmarkBase has already been annotated with @Warmup(iterations = 10).

Reply:
sorry, I was out yesterday. But my point was that even with a warm up, some volatility in the metrics can happen. Therefore, one would run the operation in question multiple times and get the average instead of a single operation. 

I haven't fully understood here, why this is not necessary in this specific case. :thinking:

Reply:
@XComp Agree that we need to run it more times to mitigate volatility. But from my understanding, we need to increase the iterations of `Measurement` instead of `[OperationsPerInvocation]`.

Reply:
And where was that done?

Reply:
IIUC, all the benchmark will inherit the `@Measurement(iterations = 10)` from `BenchmarkBase`. Do you mean we need to enlarge this value in `HighAvailabilityServiceBenchmark`? I believe we can observe for a period of time. If the results fluctuate significantly, we can consider increasing this value. WDYT?

Reply:
Ok, I did a bit more research on the difference of `@Measurement` and `@OperationPerIteration` annotations. I missed as well that we're setting `@Measurement` in the `BenchmarkBase`. Thanks for clarification. I guess, the setup is reasonable :+1: